store/sqlstore/session_store.go

```diff
@@ -48,29 +48,19 @@ func (me SqlSessionStore) Save(session *model.Session) (*model.Session, *model.A
 	if len(session.Id) > 0 {
 		return nil, model.NewAppError("SqlSessionStore.Save", "store.sql_session.save.existing.app_error", nil, "id="+session.Id, http.StatusBadRequest)
 	}
-
 	session.PreSave()
 
-	tcs := make(chan store.StoreResult, 1)
-	go func() {
-		teams, err := me.Team().GetTeamsForUser(session.UserId)
-		tcs <- store.StoreResult{Data: teams, Err: err}
-		close(tcs)
-	}()
-
 	if err := me.GetMaster().Insert(session); err != nil {
 		return nil, model.NewAppError("SqlSessionStore.Save", "store.sql_session.save.app_error", nil, "id="+session.Id+", "+err.Error(), http.StatusInternalServerError)
 	}
 
-	rtcs := <-tcs
-
-	if rtcs.Err != nil {
-		return nil, model.NewAppError("SqlSessionStore.Save", "store.sql_session.save.app_error", nil, "id="+session.Id+", "+rtcs.Err.Error(), http.StatusInternalServerError)
+	teams, err := me.Team().GetTeamsForUser(session.UserId)
```

Comment:
this variable should be `teamMebers` to be more correct.

Reply:
done